review.md

```diff
@@ -2,48 +2,73 @@
+
+## How review works
+
+When it reviews a [new pull request](https://github.com/r-multiverse/contributions/pulls), the bot makes one of three choices:
+
+1. Merge the pull request to accept the contribution.
+2. Close the pull request to reject the contribution.
+3. Flag the pull request for manual review by a [moderator](https://r-multiverse.org/governance.html#moderator).
+
+## Automatic acceptance
+
+The bot automatically accepts the contribution if the [pull request](https://github.com/r-multiverse/contributions/pulls):
+
+1. Was [created by the GitHub web interface](https://r-multiverse.org/contributors.html).
+1. Adds new contributed listings to the [`packages` folder](https://github.com/r-multiverse/contributions/tree/main/packages).
+1. Does not add, modify, or delete any other files in <https://github.com/r-multiverse/contributions>.
+
+and if each new [contributed listing](https://github.com/r-multiverse/contributions/tree/main/packages):
+
+1. Is a single line of text with a valid HTTPS URL.
+1. Points to an existing public GitHub/GitLab repository.
+
+and if the contributed GitHub/GitLab repository:
+
+1. Includes an R package at the top level whose package name is the same as the repository name.
+1. Includes a license from the "Recommended licenses" section at the end of this policy.
+1. Includes a GitHub/GitLab [release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).
+1. Is not part of the CRAN mirror at <https://github.com/cran>.
+1. Is mentioned in the `URL` field its CRAN page (if on CRAN).
+
+and if the author of the [pull request](https://github.com/r-multiverse/contributions/pulls):
+
+1. Is a [public member](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) of one of the trusted [GitHub organizations](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations) listed at <https://github.com/r-multiverse/contributions/blob/main/organizations>.
+
+In addition, the GitHub/GitLab repository owner must consent to contribute the repository to R-multiverse. Any of the following is sufficient evidence of consent:
+
+1. The author of the [pull request](https://github.com/r-multiverse/contributions/pulls) is the GitHub owner of the repository.
+1. The author of the [pull request](https://github.com/r-multiverse/contributions/pulls) is a [public member](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) of the GitHub organization that owns the repository.
```

Comment:
I'm now tempted to wonder whether that criterium is not strict enough for an organization like rOpenSci with many members and many different projects

Reply:
Good point, removed in https://github.com/wlandau/r-multiverse.github.io/commit/95dd3238a96f4fe4eb6bf7be8ea5b82599fc8613. I think this is fine because I expect the majority of packages will come from individual users, not orgs. And for orgs, it's not too much to ask to put an R-multiverse URL in the `DESCRIPTION`.

Reply:
Actually, as @shikokuchuo pointed out in https://github.com/r-multiverse/help/issues/87#issuecomment-2482364546, this question legally comes down to the license. I attempted to make this clear in https://github.com/wlandau/r-multiverse.github.io/commit/6650d559ac6d154f968c778ddf917048f34b7562. And I like how it simplifies the process.